packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -308,15 +320,27 @@
                     data-term="{{ ::'domain_dashboard_dnssec' | translate }}"
                 >
                     <oui-tile-description>
+                        <oui-switch
+                            class="mt-2 d-inline-block"
+                            data-disabled="$ctrl.vm.dnssec.isUnavailable"
```

Comment:
You can remove the `data-` here

Reply:
Hmm this one should be removed ?  (https://github.com/ovh/manager/pull/2188#discussion_r375674733 ?)

Reply:
I think I'm victim of a bug from Github because my comments don't match the line

`data-`should only be kept:
- if not on a component
- if on an attribute redefined by `ui-kit` and that already existed for `HTML`